appcelerator/titanium_mobile
# chore(ios): Un-deprecated "Ti.UI.TabbedBar" and deprecated "Ti.UI.iOS.TabbedBar"

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26674

**Summary:**
- Note that `Ti.UI.TabbedBar` support is being added to Android in 8.0.0.
- Re-added `Ti.UI.TabbedBar` support to iOS.
- Calling `Ti.UI.iOS.createTabbedBar()` now logs a deprecation warning.
- Unit test and API docs will be updated by PR #10286.

**Test:**
1. Build and run the below code on iOS.
2. Verify that a 3 button `TabbedBar` appears on screen.
3. Verify that center button of `TabbedBar` is selected/highlighted.
4. In the below code, comment out `Ti.UI.createTabbedBar()` and uncomment `Ti.UI.iOS.createTabbedBar()`.
5. Rebuild for iOS.
6. Verify that a 3 button `TabbedBar` appears on screen.
7. Verify that the following deprecation warning was logged.
`Ti.UI.iOS.TabbedBar DEPRECATED in 8.0.0, in favor of Ti.UI.TabbedBar (parity with Android)`

```javascript
function createTabbedBar(settings) {
	return Ti.UI.createTabbedBar(settings);
//	return Ti.UI.iOS.createTabbedBar(settings);
}
var window = Ti.UI.createWindow();
window.add(createTabbedBar({
	labels: ["One", "Two", "Three"],
	index: 1,
}));
window.open();
````
